ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using ntbs_service.DataAccess;
+using ntbs_service.Models;
+using ntbs_service.Pages;
+using ntbs_service.Services;
+
+namespace ntbs_service.Pages_Notifications
+{
+    public class EpisodeModel : ValidationModel
+    {
+        private readonly INotificationService service;
+        private readonly NtbsContext _context;

```

Comment:
Looks like we've been dropping the `_` from front of variables - should we standardise here, too?